src/chapters/control-flow/studio.rst

```diff
@@ -0,0 +1,113 @@
+Studio: Control Flow to Count Characters
+=========================================
+
+In this studio, you will write a program to count the number of times each
+character occurs in a string and then print the results to the console.
+
+You could prompt the user for a string, but for the sake of simplicity, you can
+start by hard-coding the string and storing it in a variable. Here’s a test
+string, for your convenience:
+
+   TODO: Decide which of these to use.
+
+   Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc accumsan sem
+   ut ligula scelerisque sollicitudin. Ut at sagittis augue. Praesent quis
+   rhoncus justo. Aliquam erat volutpat. Donec sit amet suscipit metus, non
+   lobortis massa. Vestibulum augue ex, dapibus ac suscipit vel, volutpat eget
+   massa. Donec nec velit non ligula efficitur luctus.
+
+   Twas brillig, and the slithy toves
+   Did gyre and gimble in the wabe:
+   All mimsy were the borogoves,
+   And the mome raths outgrabe.
+   Beware the Jabberwock, my son!
+   The jaws that bite, the claws that catch!
+   Beware the Jubjub bird, and shun
+   The frumious Bandersnatch!
+
+There are multiple ways to approach this task, but one way involves the
```

Comment:
How do you feel about moving this down to under the next section 'Some Items to Ponder'? The tips are related and this would hopefully deter some from jumping into things without reading the other things to ponder

Reply:
Done.